setup.py

```diff
@@ -5,26 +5,32 @@
 import sys
 from collections import defaultdict
 import logging
+
+logging.basicConfig(level=logging.INFO)
 from shutil import copytree
-from packaging.version import Version, parse as parse_version
+
+try:
+    from packaging.version import Version, parse as parse_version  # may fail with mingw
+except ImportError:
+    from pip._vendor.packaging.version import Version, parse as parse_version
+
 from setuptools import Command, Extension
 from setuptools import setup
 
+sys.setrecursionlimit(1500)
```

Comment:
Why is this needed? I think a comment here would be nice.

Reply:
Artifact, will remvoe